bentoml/cli/bento.py

```diff
@@ -74,35 +75,56 @@ def _print_bento_table(bentos, wide=False):
     _echo(table_display)
 
 
-def _print_bentos_info(bentos, output_type):
+def _print_bentos_info(bentos, output_type, print_location=False):
     if output_type == 'table':
         _print_bento_table(bentos)
     elif output_type == 'wide':
         _print_bento_table(bentos, wide=True)
     else:
         for bento in bentos:
+            if print_location:
+                _echo(bento.uri.uri)
             _print_bento_info(bento, output_type)
 
 
+def _unpack_jq_like_string(bento_dict, jq):
+    tokens = jq.split(".")
+    for _t in tokens:
+        try:
+            bento_dict = bento_dict[_t]
+        except KeyError:
+            _echo(
+                f'Key {_t} not found, ignoring', CLI_COLOR_ERROR,
```

Comment:
We can make this error message more descriptive to better inform the users. `Failed to query {json_query}. Key {_t} is missing.`